test/e2e_node/jenkins/OWNERS

```diff
@@ -0,0 +1,3 @@
+reviewers:
+- sig-node-reviewers
+- krzyzacy
```

Comment:
Would it help to add a approver section with a smaller set of approvers?

Reply:
I only try to subscribe to some notifications when people are touching those config/property files so I added myself as a reviewer for now. If you think approves are better I can make the change :-)

Reply:
As a reviewer, you'd get only a small number of PRs.

I think it makes sense to add an approver section, and put a couple of people who are aware of the situation there.